src/ott/geometry/costs.py

```diff
@@ -36,11 +36,10 @@ class CostFn(abc.ABC):
   that function is split into two norms -- evaluated on each input separately --
   followed by a pairwise cost that involves both inputs, as in:
 
-  .. math::
-
-    c(x,y) = norm(x) + norm(y) + pairwise(x,y)
+  `c(x,y) = norm(x) + norm(y) + pairwise(x,y)`
 
-  If the norm function is not implemented, that value is handled as a 0.
+  If the `norm` function is not implemented, that value is handled as a 0, and
```

Comment:
Think this can be either :func:\`norm\` or :attr:\`norm\`, not sure which would render (think the latter).

Reply:
thanks